src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/WriteStlFile.cpp

```diff
@@ -30,14 +31,6 @@ int32_t writeHeader(FILE* f, const std::string& header, int32_t triCount)
   fwrite(&triCount, 1, 4, f);
   return 0;
 }
-
```

Comment:
Did this bit of code end up being redundant

Reply:
No, I don't think its redundant, but it could certainly be inline. Ill tack it on as a small update.